app/models/external-link-service.ts

```diff
@@ -0,0 +1,48 @@
+import { attr } from '@ember-data/model';
+
+import ExternalServiceModel from './external-service';
+
+export enum SupportedResourceTypes {
```

Comment:
Should the value be camel case? e.g.

```javascript
BOOK = 'Book',
BOOK_CHAPTER = 'BookChapter',
```

, since Datacite defines resource types this way:

* https://datacite-metadata-schema.readthedocs.io/en/4.6/appendices/appendix-1/resourceTypeGeneral/

Reply:
My question is, how does the BE return those value? The enum defined here merely reflects how GV return those values, and I think [GV return those values in `UPPER_CASE`](https://github.com/CenterForOpenScience/gravyvalet/blob/develop/addon_service/common/enum_serializers.py#L64).

Reply:
fyi, the ticket https://openscience.atlassian.net/browse/ENG-7885